capture-traffic/capture-network-traffic.md

```diff
@@ -45,15 +45,12 @@ The **Network Capture** mode is in Beta state and, currently, has some known lim
 
 - The network capturing mode captures IPv4 traffic, while IPv6 is not yet supported.
 
-- The network capturing mode is incompatible with most VPN tools.
+- Some VPN tools might close the VPN connection if a third-party network extension is enabled. You can try to workaround that by making the VPN connection **before** starting Fiddler Everywhere and then enabling its network capturing mode.
 
-    >tip Some VPN tools will automatically close the VPN connection if a third-party network extension is enabled. You can try to workaround that by making the VPN connection before starting Fiddler Everywhere and enabling the network capturing mode.
-
-- The **HTTPS > Connections > Bypass Fiddler for URLs that starts with:** option is incompatible with the **Network Capture** mode. Alternatively, you can create a Fiddler rule that executes the **Do Not Decrypt** action on the desired domains.
+    >tip To use a VPN connection alongside Fiddler you often need to bypass specific VPN endpoints. However note that The **HTTPS > Connections > Bypass Fiddler for URLs that starts with:** option is incompatible with the **Network Capture** mode. Alternatively, you can bypass targeted domains [by creating a Fiddler rule that executes the **Do Not Decrypt** action]({%slug configure-vpn-fiddler%}#configuration-for-network-capturing-mode).
```

Comment:
I think we need to clarify further when is the bypass step required - is it specific to a case when the VPN doesn't close the connection or it is a requirement in all cases.